gc3libs/config.py

```diff
@@ -218,13 +218,30 @@ def __init__(self, *locations, **extra_args):
         # save the list of (valid) config files
         self.cfgfiles = []
 
+        if not cfg_dict is None:
```

Comment:
This would be more idiomatic Python:
```suggestion
        if cfg_dict is not None:
```

but I would actually just use the implicit test here, since we don't care about distinguishing when `cfg_dict` is `None` or the empty dict:

```suggestion
        if cfg_dict:
```